### PR TITLE
Clinical slides outline

### PR DESCRIPTION
**Pull request by robinandeer**
I'll write this part intro sections that represent slides in the presentation. Then I will replicate that in Keynote.

I guess it could be a good idea to create a separate Keynote file for this part of the presentation and merge at the end. I'll make some folder reorgs.

EDIT: I should've opened a new PR perhaps for the reorg. but perhaps merge this before you start working on you part then instead :smiley:

### CODE REVIEW

**Comment by robinandeer** [comment]
OK, so I added a template for Keynote that we can use as starting point for each of the individual parts - just make a copy into the part-folder and hack away!

File: parts/3-clinical/slides.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+means processing the data as far as possible and intuitively visualize it.
+
+### Mission
+5 min analysis
+
+
+## Genomics in healthcare today
+In fact, clinics already routinely send in patient samples for sequencing.
+This is pretty cool because healthcare is *not* famous for moving fast or
+adopting new technology.
```
Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: oh, so is the same in Sweden... :sweat_smile:

File: parts/3-clinical/slides.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+   "waste" that no ones has figured out a plan to do with. Kind of like
+   a nuclear power plant.
+
+As the patient sample travels through our facility thousands of lines of
+Python code will eventually touch it. I'm going to focus on the delivery
+report.
+
+
+## Excel sheet -> Web interface
+What these guys [point to Guillermo] are delivering to researchers is very
```
Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: hmm that's not completely true, we do the analysis and what we give back are sample and project reports (and raw data). It is true though that as far as I know most of the time researchers just re-run our analysis :-/ but they don't have to

Comment (robinandeer) [inline parts/3-clinical/slides.md]: sure that's of course true (both things)

But I'm trying to think how much it matters to the people in the audience? We'll have to make generalizations to keep it interesting even if it doesn't reflect reality completely, right? :smiley:

Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: right :)

**Pull request merged**